interface/wx/intl.h

```diff
@@ -400,7 +400,7 @@ class wxLocale
         This function may be used to find the language description structure for the
         given locale, specified either as a two letter ISO language code (for example,
         "pt"), a language code followed by the country code ("pt_BR") or a full, human
-        readable, language description ("Portuguese-Brazil").
+        readable, language description ("Portuguese_Brazil").
```

Comment:
Does this mean "Portuguese-Brazil" doesn't and won't work? I think that ideally both it and the version with the underscore should work.

Reply:
> Does this mean "Portuguese-Brazil" doesn't and won't work? I think that ideally both it and the version with the underscore should work.

Right, "Portuguese-Brazil" won't work. And I don't intend to change that - at least at the moment, unless you convince me to do otherwise. The reason is simple: there are a few regions that have a hyphen in their name, for example "Portuguese_Guinea-Bissau" or "Portuguese_Timor-Leste". And theoretically the language name could also be a composed name with a hyphen (although we currently don't have such an entry in our language database).

If you insist to support "Portuguese-Brazil" as well, we could do so by first checking for an underscore separator, and - if not found - for a hyphen in a second step.

Reply:
In the light of the fact, that this form of specifying the language never worked in previous wx versions, it seems reasonable to restrict the syntax to an underscore separator. If there is a demand, this can be changed later. Nevertheless, a note about the allowed separator probably makes sense, because in the documentation of previous wx versions we have still the sample with a hyphen separator.